partiql-parser/src/scanner.rs

```diff
@@ -68,8 +72,8 @@ pub struct Token<'val> {
 
 impl<'val> Token<'val> {
     /// Returns the parsed content of this token.
-    pub fn content(&self) -> Content<'val> {
-        self.content
+    pub fn content(&self) -> &Content<'val> {
+        &self.content
     }
```

Comment:
Can't return by value anymore because the `Cow` makes `Content` potentially expensive to do so.